### PR TITLE
fix(ci): do not ask for upgrade on CI

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

Fixes https://linear.app/nango/issue/NAN-1818/do-not-ask-for-upgrade-cli-on-ci

- CLI: do not ask for upgrade on CI

### CODE REVIEW

**Review by TBonnin — APPROVED** [review]

**Comment by khaliqgant** [comment]
Why won’t the NANGO_CLI_UPGRADE env variable work here?

File: packages/cli/lib/index.ts [hunk 1]
```diff
@@ -36,7 +36,9 @@ class NangoCommand extends Command {
                 }
             }
 
-            await upgradeAction(debug);
+            if (isCI) {
```
Comment (khaliqgant) [inline packages/cli/lib/index.ts]: Don’t we want the upgrade action if it is not CI?

Comment (bodinsamuel) [reply packages/cli/lib/index.ts]: yep indeed 🤦🏻

**Comment by bodinsamuel** [comment]
> Why won’t the NANGO_CLI_UPGRADE env variable work here?

It would for sure, it's just unnecessary to ask for customers to know about it

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**